src/sentry/receivers/useremail.py

```diff
@@ -19,3 +21,20 @@ def create_user_email(instance, created, **kwargs):
     dispatch_uid="create_user_email",
     weak=False
 )
+
+
+@email_verified.connect(weak=False)
+def verify_newsletter_subscription(email, **kwargs):
+    from sentry.app import newsletter
+
+    if not newsletter.enabled:
+        return
+
+    if not email.is_primary():
+        return
+
+    newsletter.update_subscription(
+        email.user,
```

Comment:
Oh, right, I want to grab the `sender`. I forgot this is 100% passed by kwargs. GOOD CATCH.